### PR TITLE
Associate BundleInstances with existing Application Instances

### PR DESCRIPTION
**Pull request by nick-benoit14**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -175,6 +175,13 @@ def setup_application_instances(application, application_instances)
   end
 end
 
+BundleInstance.all.each do |bundle_instance|
```
Comment (bfcoder) [inline db/seeds.rb]: Always use `.find_each` instead. Doing `.all.each` will load every single record into memory. Which isn't bad when you are local, but as soon as you have 100,000 records on prod...that really hurts and will likely cause this to blow up. `.find_each` will, by default, load 1,000 records at a time.

Comment (nick-benoit14) [inline db/seeds.rb]: Good to know, thanks

File: db/seeds.rb [hunk 2]
```diff
@@ -175,6 +175,13 @@ def setup_application_instances(application, application_instances)
...
+  bundle_instance.applications.each do |app|
+    instance = ApplicationInstance.find_by(site: bundle_instance.site, application: app)
```
Comment (bfcoder) [inline db/seeds.rb]: consider storing the site to a variable outside this `each`
also could write this as
```ruby
instance = app.application_instances.find_by(site: site)
```

File: db/seeds.rb [hunk 3]
```diff
@@ -175,6 +175,13 @@ def setup_application_instances(application, application_instances)
   end
 end
 
+BundleInstance.find_each do |bundle_instance|
+  bundle_instance.applications.each do |app|
```
Comment (bfcoder) [inline db/seeds.rb]: `.find_each` here too.

File: db/seeds.rb [hunk 4]
```diff
@@ -175,6 +175,13 @@ def setup_application_instances(application, application_instances)
...
+    instance = ApplicationInstance.find_by(site: bundle_instance.site, application: app)
```
Comment (bfcoder) [inline db/seeds.rb]: consider storing the site to a variable outside this `each`
also could write this as
```ruby
instance = app.application_instances.find_by(site: site)
```

File: db/seeds.rb (hunk 3 above)
Comment (bfcoder) [inline db/seeds.rb]: Isn't as big of a deal here, as I can't imagine there being more than 1,000 results. But I think it good practice to use it.

Comment (nick-benoit14) [inline db/seeds.rb]: Even though this only looks through all of the applications associated with bundle_instance? If we have more than 1000 applications associated with a bundle_instance we might have bigger problems.

Comment (bfcoder) [inline db/seeds.rb]: that is true. you decide if you want to use it. :]

Comment (bfcoder) [inline db/seeds.rb]: All it really does is add `LIMIT 1000` to the sql.

**Pull request merged**